.github/workflows/performance-test.yml

```
@@ -0,0 +1,516 @@
+
+    - name: Disable ufw
+      run: |
+        sudo ufw disable
+
+    - name: Download test-image-pr
+      uses: actions/download-artifact@v4
+      with:
+        name: test-image-pr
+
+    - name: Load container image
+      run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:21: Double quote to prevent globbing and word splitting

(shellcheck)